src/Monocle/Main.hs

```diff
@@ -158,7 +158,8 @@ run' ApiConfig {..} aplogger = E.runConcurrent $ runLoggerEffect do
 
     let settings = Warp.setPort port $ Warp.setLogger aplogger Warp.defaultSettings
         jwtCfg = localJWTSettings
-        cfg = jwtCfg :. defaultCookieSettings :. EmptyContext
+        cookieCfg = defaultCookieSettings {cookieXsrfSetting = Nothing}
```

Comment:
I think that's ok to do since we don't have POST form or anything that could be maliciously manipulated, though it would be good to know why this is not working.

Reply:
Agreed. My theory about the potential cause is explained in the commit message. Perhaps using session in Axios might solve the issue. I did some research to see if other are affected by that behavior but I found nothing relevant. I'll be happy to show you the debug console of the browser ... then if you get any idea about how we could fix this (though if it worth the try)